cypress/objects/loginPage.js

```diff
@@ -0,0 +1,60 @@
+class LoginPage {
+    visit() {
+        cy.visit('/');
+    }
+
+    getUserTitle() {
+        return cy.get('.user-info__title');
+    }
+
+    getURL() {
+        return cy.url();
```

Comment:
this function is kind of redundant, it just calls another function with no parameters